javascript/broadcast_from_element.js

```diff
@@ -0,0 +1,57 @@
+import morphdom from 'morphdom'
+import { consumer } from './action_cable'
+
+class BroadcastFromElement extends HTMLElement {
+  connectedCallback () {
+    if (this.preview) return
+    if (consumer) {
+      this.channel = consumer.subscriptions.create(
+        {
+          channel: 'CableReady::Stream',
+          identifier: this.getAttribute('identifier')
+        },
+        {
+          received: () => {
+            const identifier = this.getAttribute('identifier')
+            const query = `broadcast-from[identifier="${identifier}"]`
+            const blocks = document.querySelectorAll(query)
+            if (blocks[0] !== this) return
+
+            const template = document.createElement('template')
+            fetch(
+              this.hasAttribute('data-url')
```

Comment:
Being a custom element, this could just be `url`.

On custom elements I prefer not to overuse the dataset but make everything normal attributes

Reply:
It started off as `url` but @andrewculver suggested `data-url` would make it more compatible with Turbo Frames (I'm paraphrasing!)

Is this not the case?

Reply:
Might be! Hasn't occurred to me

Reply:
@leastbad @julianrubisch I agree with you guys now, it should just be `url`. Won't even bother trying to rehash what I was talking about before, because my thinking on the whole thing has changed.

Reply:
@leastbad @julianrubisch I agree with you two now, it should just be url. Won't even bother trying to rehash what I was talking about before, because my thinking on the whole thing has changed.

Reply:
NP! Updated to `url` from `data-url`.